### ISSUE TITLE
Categorical and Continuous Test Results

### ISSUE THREAD

**Issue by giannispic** [Tandon-A/emotic#1]
Hello. I was wondering how did your models perform in the emotic test set. Did you manage to match or even surpass the author's original results?

**Comment by Tandon-A** [comment]
@giannispic 
Hello, 

I apologize for the late reply. 

I wasn't able to match the author's result. The maximum mean AP that I achieve over the test set is 26.02, about 1.3 points lower than what the author's report. 
Do note that I am using ResNet18 models as feature extractors, whereas they are using [Decomposeme](https://arxiv.org/abs/1606.05426) based models. These pre-trained models aren't available for PyTorch. I had experimented with models other than ResNet18s, but that resulted in no significant improvement. 

Things that worked for me: 
* Changing the batch size. The author had used a batch size of 52, but I found a batch of 26 images to work better.  
* Increasing the depth and the features (neurons in each layer) of the fusion network.
* Updating the weights of only the last layer of the feature extractor as compared to training the whole feature-extractor yielded better performance. 

You might also be interested in looking into [EmotionNet](https://openaccess.thecvf.com/content_CVPR_2020/papers/Wei_Learning_Visual_Emotion_Representations_From_Web_Data_CVPR_2020_paper.pdf) and use that model as the feature extractor. 

Regards, 
Abhishek
